python-sdk/nuscenes/eval/panoptic/evaluate.py

```diff
@@ -1,9 +1,9 @@
 """
 Code written by Motional and the Robot Learning Lab, University of Freiburg.
 
-Script to evaluate nuScenes-panoptic panoptic segmentation (PS) or multi-object panoptic tracking metrics (MOPT).
+Script to evaluate Panoptic nuScenes panoptic segmentation (PS) or panoptic tracking (PT) metrics.
 Argument "task" could be one of ["segmentation", "tracking"], check eval/panoptic/README.md for more details of the
-tasks. Note tracking results will be evaluated with both the MOPT and PS metrics.
+tasks. Note tracking results will be evaluated with both the PT and PS metrics.
```

Comment:
PT -> PAT?

Reply:
PT stands for panoptic tracking, PS for panoptic segmentation, will use the full names.